### PR TITLE
Home page footer #17

### PR DESCRIPTION
**Pull request by Kangonga**
# Description
This pull request is part of the work to create a footer for the homepage. 
To achieve this, I needed to:

```

- Create the  call to action component and import it into its direct parent module.
- Import the buttons module to access the animated button component 
- Copy the data for the footer ie the HTML data and style using scss.
- Make it visible on the homepage by nesting the call to action component within the elewa footer component
```

Fixes # ([17](https://github.com/italanta/elewa-group/issues/17))

## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Screenshot (optional)
PC view
Screenshot from 2023-02-17 22-52-41

Mobile view
Screenshot from 2023-02-17 22-53-17

# How Has This Been Tested?

Using chrome developer tools, to ensure responsiveness in multiple screen sizes

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Implement #252 first then come back to this.

File: libs/pages/elewa/home/src/lib/components/cta-mission/cta-mission.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="footer-content">
+    <h4>do you want to be part of our mission?</h4>
+    <div class="mission">
+      <elewa-group-button-with-animation [message]="'Become an investor'" [mode]="'dark'" [action]="'/'"></elewa-group-button-with-animation>
+    </div>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/components/cta-mission/cta-mission.component.html:5]: should be a reusable component

File: libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html [hunk 2]
```diff
@@ -6,6 +6,9 @@
       <elewa-group-activity-section></elewa-group-activity-section>
       <elewa-group-team-and-partners></elewa-group-team-and-partners>
       <elewa-group-home-jobs-section></elewa-group-home-jobs-section>
+      <elewa-group-footer>
+        <elewa-group-cta-mission footer ></elewa-group-cta-mission>
+      </elewa-group-footer>
     </div>
   </elewa-group-elewa-group-main-page>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/home/src/lib/pages/home-page/home-page.component.html:13]: footer is added on main-page only.

Check #252 first

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Closing this to aid development of dependent issue first

**Pull request closed without merging**